src/features/events/hooks/useEventClusterData.ts

```diff
@@ -0,0 +1,131 @@
+import { ClusteredEvent } from 'features/campaigns/hooks/useClusteredActivities';
+import { EnvContext } from 'core/env/EnvContext';
+import { EventState } from 'features/events/models/EventDataModel';
+import getEventStats from 'features/events/rpc/getEventStats';
+import { loadItemIfNecessary } from 'core/caching/cacheUtils';
+import { RootState } from 'core/store';
+import { STATUS_COLORS } from 'features/campaigns/components/ActivityList/items/ActivityListItem';
+import { useContext } from 'react';
+import { useStore } from 'react-redux';
+import { ZetkinEvent } from 'utils/types/zetkin';
+import { statsLoad, statsLoaded } from 'features/events/store';
+
+export default function useEventClusterData(cluster: ClusteredEvent) {
```

Comment:
very interesting test. looking forward to seeing what more this can bring to what we use.